dhis-2/dhis-web-api/src/main/resources/openapi/EnrollmentsExportController.md

```diff
@@ -6,58 +6,66 @@ Get enrollments matching given query parameters.
 
 ### `getEnrollments.parameter.enrolledAfter`
 
-Get enrollments enrolled after given date.
+Get enrollments with an enrollment date after the given date.
 
 ### `getEnrollments.parameter.enrolledBefore`
 
-Get enrollments enrolled before given date.
+Get enrollments with an enrollment date before the given date.
 
 ### `getEnrollments.parameter.enrollment`
 
 `<enrollment1-uid>[;<enrollment2-uid>...]`
 
-Get enrollments with given UID(s).
+Get enrollments with the given UID(s).
 
 ### `getEnrollments.parameter.followUp`
 
-Get enrollments with given follow-up status of the instance for the given program.
+Get enrollments with the given follow-up status.
 
 ### `getEnrollments.parameter.includeDeleted`
 
 Get soft-deleted enrollments by specifying `includeDeleted=true`. Soft-deleted enrollments are excluded by default.
 
 ### `getEnrollments.parameter.orgUnit`
 
-Get enrollments owned by given `orgUnit`.
+Get enrollments owned by given `orgUnit` relative to the `ouMode`. 
+- When `ouMode=SELECTED` - or no `ouMode` is given - the enrollments owned by the `orgUnit` is returned.
+- When `ouMode=CHILDREN` the enrollments owned by the `orgUnit` or by the `orgUnit`s direct children is returned.
+- When `ouMode=DESCENDANTS` the enrollments owned by the `orgUnit` or by any of the descendants under the `orgUnit` is returned.
+- When `ouMode=ALL`, `ouMode=CAPTURE` or `ouMode=ACCESSIBLE` the `orgUnit` parameter will be ignored.
 
 ### `getEnrollments.parameter.ouMode`
 
 Get enrollments using given organisation unit selection mode.
+- When `ouMode=SELECTED`, `ouMode=CHILDREN` or `ouMode=DESCENDANTS`, the `orgUnit` parameter is also required to specify which enrollments to download.
```

Comment:
```suggestion
- When `ouMode=SELECTED`, `ouMode=CHILDREN` or `ouMode=DESCENDANTS`, the `orgUnit` parameter is required to specify which enrollments to return.
```

I would stick to `returned` instead of `downloaded` 😄